docs/tutorial/index.rst

```diff
@@ -268,3 +281,6 @@ If you click on the "Details" link while it is building,
 you will access the build logs,
 otherwise it will take you directly to the documentation.
 When you are satisfied, you can merge the pull request!
+
+That's the end of the tutorial for now,
```

Comment:
`for now` feels weird, but I don't have a great suggestion.

Reply:
Let's just remove it?